seqr/utils/elasticsearch/es_search.py

```diff
@@ -65,7 +65,7 @@ def __init__(self, families, previous_search_results=None, skip_unaffected_famil
             if len(self.samples_by_family_index) < 1:
                 raise Exception('Inheritance based search is disabled in families with no affected individuals')
 
-        self._indices = sorted(list(self.samples_by_family_index.keys()), reverse = True)
+        self._indices = sorted(list(self.samples_by_family_index.keys()))
```

Comment:
The edge case we are testing is if different indices have different subsets of individuals loaded for a family, which is true in our test data for family 2: `test_index_sv` has individuals 4 and 5, and `test_index` has individuals 4, 5, and 6. However, because of the iteration order `test_index` was updated last and since it has the superset there were no errors. By reversing the iteration order here so the index with fewer samples is looked at last we duplicate the production error, and a couple of test start to fail with the same key error we see in the original bug report. Since we only added the sorting here for test stability in the first place, there are no negative consequences for swapping the order, and now going forward that edge case will have a unit test so we don;t reintroduce the bug

Reply:
Can we combine the individuals of the family from different indices instead of overwriting those from the previous index?